optimizely/client.py

```diff
@@ -8,15 +8,21 @@
 
 BASE_URL = 'https://www.optimizelyapis.com/experiment/v1/'
 
+VALID_TOKEN_TYPES = ('legacy', 'oauth')
 
 class Client(object):
   ALLOWED_REQUESTS = ['get', 'post', 'put', 'delete']
 
-  def __init__(self, api_key, api_base=BASE_URL):
+  def __init__(self, api_key, token_type, api_base=BASE_URL):
     # set API information
     self.api_key = api_key
     self.api_base = api_base
 
+    if token_type in VALID_TOKEN_TYPES:
+      self.token_type = token_type
+    else:
+      raise ValueError('Invalid Token Type')
```

Comment:
would be good to enumerate the valid token types in this error. just something like raise ValueError('Invalid token type. Valid types are: %s' % (VALID_TOKEN_TYPES))